content/tutorials/mushr_navigation_system.md

```diff
@@ -0,0 +1,167 @@
+---
+title: "MuSHR multi-agent navigation system tutorial"
+date: 2021-04-18T22:17:25+05:30
+summary: "This tutorial covers running the MuSHR multi-agent navigation stack in simulation"
+difficulty: "Advanced"
+duration: 30
+featured: false  # whether this is listed at / (must also be top 6 by weight). 
+active: true     # whether this is listed at /tutorials/
+draft: true      # whether Hugo considers this a draft
+weight: 3        # 2 = intro tutorial 3 = anything else
+---
+
+<h2> By: <a href=https://www.sidharthtalia.com/>Sidharth Talia</a></h2>
+
+<!-- Header figure required! -->
+<br>
+{{< figure src="/tutorials/MuSHR_multiagent_navigation/intro.jpg" width="800" >}} <br>                           
+<br>
+
+### Introduction
+A navigation system enables a robot to navigate quickly between different poses while avoiding collisions with the environment or other agents. The navigation system in this project uses the NH-TTC [backend](https://github.com/davisbo/NHTTC), which is a decentralized multi-agent navigation system. The Non-holonomic-Time-To-Collision (NH-TTC) backend considers the Non-holonomic constraints of the car and considers the time to collision with other cars in the optimization process. To learn more about how it works, checkout the paper [here](http://motion.cs.umn.edu/r/NH-TTC/arxiv-NHTTC.pdf)!
+
+### Goal
+The goal of this tutorial is to get the multi-agent navigation system up and running on your system.
+
+### Requirements
+Completed the [quickstart](https://mushr.io/tutorials/quickstart/) tutorial (familiarity with ROS and python is assumed).
+
+## Environment setup
+Clone the nhttc_ros repository into your catkin workspace: 
+{{< highlight bash >}}
+$ cd catkin_ws/src
+$ git clone --branch devel https://github.com/naughtyStark/nhttc_ros.git
```

Comment:
we can, depending on when the tutorial actually gets published. If it is published right now, it should be devel because the devel branch is the most up-to-date and stable (as far as sim is concerned).

Reply:
Why not push devel to main if it is stable?